src/components/About.js

```diff
@@ -1,9 +1,23 @@
 import React from "react";
+import Image from './Image'
+import brushes from './assets/brushes.jpg'
+import recycle from './assets/recycle.jpg'
+import palette from './assets/palette.jpg'
 
 export default function About() {
   return (
-    <div>
-      <h1>About Makeup Showroom</h1>
+    <div className="about-container">
+      <h1 style={{marginBottom: '2rem'}}>About Makeup Showroom</h1>
+      <div className="about-section">

```

Comment:
@pasternakewa if You have repeated partes, I prefer just create an object and iterate through it

Reply:
> done